chanfitness/chanevent.go

```diff
@@ -79,24 +107,66 @@ func (e *chanEventLog) close() {
 	e.closedAt = e.now()
 }
 
+// listEvents returns all of the events that our event log has tracked,
+// including events that are staged for addition to our set of events but have
+// not yet been committed to (because we rate limit and store only the aggregate
+// outcome over a period).
+func (e *chanEventLog) listEvents() []*channelEvent {
```

Comment:
Indeed, this would draw the line between event log and store more cleanly in the unit tests, which is nice. Got a few PRs coming up in this area so will keep it in mind!

Reply:
👍